schemas.js

```diff
@@ -24,12 +24,38 @@ const extension = (joi) => ({
 
 const Joi = BaseJoi.extend(extension);
 
-module.exports.campgroundSchema = Joi.object({
-  campground: Joi.object({
-    title: Joi.string().required().escapeHTML(),
-    price: Joi.number().required().min(0),
-    location: Joi.string().required().escapeHTML(),
-    description: Joi.string().required().escapeHTML(),
+module.exports.sendPlayDataSchema = Joi.object({
+  difficulty: Joi.number().max(5).required(),
+  sessionLength: Joi.number().required(),
```

Comment:
`difficulty` has `.max(5)` why not `sessionLength` as well?

Reply:
I could have added a value check for `sessionLength` as well, but figured it wasn't necessary yet, as the `sessionLength`s will be changed once the app is ready for deployment (from 3, 5, 7, 9, 11 to something like 10, 20, 50, 100, 200). Thought I'd simply add a check for the exact length values once i get there.

I probably should add a .min() value check for difficulty though.